libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -690,6 +708,16 @@ public void onToolbarFormatButtonClicked(ITextFormat format, boolean isKeyboardS
             mEditorFragmentListener.onTrackableEvent(TrackableEvent.READ_MORE_BUTTON_TAPPED);
         } else if (format.equals(CommentsTextFormat.FORMAT_PAGE)) {
             mEditorFragmentListener.onTrackableEvent(TrackableEvent.NEXT_PAGE_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_HORIZONTAL_RULE)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.HORIZONTAL_RULE_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_LEFT)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_LEFT_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_CENTER)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_CENTER_BUTTON_TAPPED);
+        } else if (format.equals(AztecTextFormat.FORMAT_ALIGN_RIGHT)) {
+            mEditorFragmentListener.onTrackableEvent(TrackableEvent.FORMAT_ALIGN_RIGHT_BUTTON_TAPPED);
+        } else {
+            AppLog.w(T.EDITOR, "Aztec ToolbarFormatButtonClick not tracked in Analytics: " + format.toString());
```

Comment:
Should we log this to Sentry to make it more visible to us?

Reply:
I guess it makes sense to add it, but Fragment is part of the Editor library, and doesn't have direct access to `CrashLoggingUtils`.